src/stores/keyblockDetails.js

```diff
@@ -13,20 +15,25 @@ export const useKeyblockDetailsStore = defineStore('keyblockDetails', () => {
     return rawKeyblock.value ? adaptKeyblock(rawKeyblock.value, keyblockDeltaStats.value) : null
   })
 
+  async function getIt(param) {
```

Comment:
This is the solution taken from old explorer

Reply:
well... considering we need to apply it on every response, maybe we should just use Axios interceptors and apply camelcaseKeysDeep globally: https://axios-http.com/docs/interceptors. If you don't like this option, another good practice I saw and used in most of the projects is creating an "API Client" which is basically an Axios wrapper object put in the root catalog or utils. This gives a little bit more flexibility compared to a global interceptor.

Reply:
Thank you, I guess Interceptor is what I wanted to use. Thanks for the hint, I will try to move this forward